examples/vision/cutmix.py

```diff
@@ -0,0 +1,383 @@
+"""
+Title: CutMix Augmentation for Image Classification
```

Comment:
Suggestion (also lowercase/sentence case):

```suggestion
Title: CutMix data augmentation for image classification
```